Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/timeline.js

```diff
@@ -393,6 +393,8 @@ class TimelineFromEndpoint {
         });
     }
     update() {
+        if (this.radarButtonRef)
```

Comment:
This should be `issueButtonRef` or something that isn't specific to radar.

Reply:
I think we will need another bugzilla button ref as well, so I make this radar button.

In my design, we will have two different button, one is for radar and one is for bugzilla

Reply:
Then I think we need a list of button refs.

Presumably, we'll be adding GitHub Issues at some point in the future.